model/ftn/ww3_ounp.ftn

```diff
@@ -2665,12 +2666,25 @@
       IRET=NF90_DEF_VAR(NCID, 'time', NF90_DOUBLE, DIMID(1), VARID(1))
       CALL CHECK_ERR(IRET)
 !/NC4      IF (NCTYPE.EQ.4) IRET=NF90_DEF_VAR_DEFLATE(NCID, VARID(1), 1, 1, DEFLATE)
-      IRET=NF90_PUT_ATT(NCID,VARID(1),'long_name','julian day (UT)')
+      IF (CAL360) THEN
+        IRET=NF90_PUT_ATT(NCID,VARID(1),'long_name','days since 1900-01-01')
```

Comment:
long_name should be described as : "All years are 360 days divided into 30 day months." regarding the CF convention 
http://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#calendar

Reply:
@mickaelaccensi Is that correct? I think that "long_name" should actually just be "time" with an appropriate value in the "calendar" attribute.

Reply:
@ukmo-ccbunney and @mickaelaccensi 

standard_name should be 'time'

long_name can be whatever we choose, but re Mikael's comment it is probably advisable if it indicates the form of calendar we are using in the non-standard cases (makes it an easy read for user, and the same should go for a noleap time variable?)

as Chris says, the important attribute here is 'calendar'; see http://cfconventions.org/cf-conventions/cf-conventions.html#calendar

there is always the option of the 'comment' attribute if we really aren't happy, but best if the variable can be made clear via standard_name, units, and calendar

So a  preferred answer might be something like:

long_name = 'days since 1900-01-01 in 360 day calendar'

As an aside to this change, the calendar attribute in ww3_ounf should also have a 'no_leap' or '365_day' option?

Reply:
@ukmo-ansaulter @mickaelaccensi 
> So a preferred answer might be something like:
> long_name = 'days since 1900-01-01 in 360 day calendar'

OK, although I think in that case the units should go in the long_name. Long name is described as:

>  "_a long descriptive name which may, for example, be used for labeling plots_",

so, to my mind, it should describe the values after they have been "decoded" by having the units applied to them.
E.g. "_time in 360 day calendar_", or similar.

Reply:
ok for long_name = "time in 360 day calendar"

Reply:
I will use the suggested long_name="time in 360 day calendar", and add long_name and calendar attributes for the no leap calendar using cf convenctions, leaving units as 'days since 1990-01-01 00:00:00' and the rest of the attributes unchanged.

Reply:
Change looks OK to me

Reply:
Looks good to me.